src/system/core/handlebars.js

```diff
@@ -193,6 +193,20 @@ function registerHandlebarsHelpers() {
     return new Handlebars.SafeString(value.replace(/(<([^>]+)>)/gi, "").replace(/(?:\r\n|\r|\n)/g, "<br/>"));
   });
 
+  Handlebars.registerHelper("feelingChoice", function (current_selection, axis, choice) {
```

Comment:
I really like how this turned out.

Reply:
Thanks! If we had access to bond within the helper, I think we could've gotten away without including the first argument, but oh well